tutorials/gcs-cf-auth/src/getToken.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * github GoogleCloudPlatform/community/tutorials/gcs-cf-auth
+ * Utility script to generate OAuth 2.0 authentication tokens
+ * See ../README.md for license and more info
+ *
+ * USAGE:
+ * This tool is designed to be executed within the the GCS Cloud Shell or a PChell that is configured to use NodeJS
+ * terminal that is configured to use NodeJS.
```

Comment:
'...terminal that is configured to use NodeJS.'

This phrase seems mostly redundant.